libraries/chain/deep_mind.cpp

```diff
@@ -83,12 +83,14 @@ namespace eosio::chain {
       );
    }
 
-   void deep_mind_handler::on_accepted_block_v2(block_num_type lib, const signed_block_ptr& b, const finality_data_t& fd)
+   void deep_mind_handler::on_accepted_block_v2(block_num_type lib, const signed_block_ptr& b, const block_id_type& id,
```

Comment:
Not a big deal. Maybe it is nicer the order of parameters matches the order of DM fields.